tests/unit/python/foglamp/services/core/api/test_configuration.py

```diff
@@ -207,16 +210,20 @@ def reset_singleton(self):
                 assert 200 == resp.status
                 r = await resp.text()
                 json_response = json.loads(r)
+                if hide_password:
+                    result['value'] = "****"
```

Comment:
why password hide is manual process here, is this not the expected result? and to be done by the executed code.

Reply:
`expected result Vs Actual result` - There is no manipulation on actual result, it's the way to modify the data setup for tests and there are n number of way to deal it.

Reply:
Got it. I missed to read. `@pytest.mark.parametrize("result` this result is expectedResult. I got it confused with manipulation on actual result.